vector/src/main/java/im/vector/riotx/features/home/room/detail/timeline/factory/MessageItemFactory.kt

```diff
@@ -241,12 +243,13 @@ class MessageItemFactory @Inject constructor(
         return MessageTextItem_()
                 .apply {
                     if (informationData.hasBeenEdited) {
-                        val spannable = annotateWithEdited(linkifiedBody, callback, informationData)
+                        val spannable = annotateWithEdited(linkifiedBody, callback, attributes.dimensionConverter, informationData)
```

Comment:
Why taking dimension converter from attributes? It should be injected in MessageItemFactory constructor instead, right?

Reply:
yes